editor.planx.uk/src/@planx/components/Notice/Public.tsx

```diff
@@ -75,6 +76,13 @@ const NoticeComponent: React.FC<Props> = (props) => {
     ? () => props.handleSubmit?.()
     : undefined;
 
+  const { trackResetFlow } = useAnalyticsTracking();
+
+  const handleNoticeResetClick = () => {
+    trackResetFlow()
+    props.resetPreview && props.resetPreview()
+  }
```

Comment:
nit: row end semi-colons in the body here would make this more readable for me? (i'm surprised linting/husky didn't pick this up actually!)

Reply:
Fixed here: ba123628fbfe319d4d1f22391c498b6e0532a155

Reply:
I re-ran `pnpm install` in `editor.planx.uk` and that installed husky and it appears to be working now. 

I'm not really sure why that was required although it's working now 🤷